org.osgi.test.junit5/test.bndrun

```diff
@@ -53,4 +53,8 @@
 	org.osgi.test.common;version='[0.9.0,0.9.1)',\
 	org.osgi.test.junit5-tests;version='[0.9.0,0.9.1)',\
 	org.osgi.test.junit5;version='[0.9.0,0.9.1)',\
-	osgi.promise;version='[6.0.0,6.0.1)'
+	osgi.promise;version='[6.0.0,6.0.1)',\
+	junit-jupiter-params;version='[5.6.2,5.6.3)',\
+	junit-platform-testkit;version='[1.6.2,1.6.3)'
+
+-runee: JavaSE-1.8
```

Comment:
Do we need this?

Reply:
No - it snuck in when I was trying to debug bndtools/bnd#4130